app/files.test.js

```diff
@@ -179,6 +180,26 @@ Files("triggers when a file is deleted", async (context) => {
   else equal(true, true)
 })
 
+Files("triggers when a user invited to Room", async (context) => {
+  // TODO: After merging with the invite-user-action branch,
+  // add a test with an array length check.
+  // If active true, an array of length 1 will be returned
+  // if active false, array will consist of two elements.
```

Comment:
If you have an idea of how to improve this test case but require a feature from another branch, you can create a new branch and merge the current one with the branch that contains the required feature. Once done, create a new pull request under draft.